website/snippets/_new-sl-setup.md

```diff
@@ -1,5 +1,9 @@
 You can set up the dbt Semantic Layer in dbt Cloud at the environment and project level. Before you begin:
+
 - You must have a dbt Cloud Team or Enterprise [multi-tenant](/docs/cloud/about-cloud/regions-ip-addresses) deployment, hosted in North America. 
+- You must be part of the Owner group, and have the correct [license](/docs/cloud/manage-access/seats-and-users) and [permissions](/docs/cloud/manage-access/self-service-permissions) to configure the Semantic Layer:
```

Comment:
I think the only thing to change here is that if you are an Account Admin, you actually don't have to be an Owner. You can just be "Everyone" in this case